src/Razor/src/Microsoft.VisualStudio.LiveShare.Razor/Host/DefaultProjectSnapshotManagerProxy.cs

```diff
@@ -109,15 +107,15 @@ internal async Task<ProjectSnapshotManagerProxyState> CalculateUpdatedStateAsync
                 foreach (var project in projects)
                 {
                     var projectHandleProxy = ConvertToProxy(project);
-                    projectHandles.Add(projectHandleProxy);
+                    projectHandles.Add(projectHandleProxy!);
```

Comment:
Suppressing the result of a private method call is odd. Can we annotate it better? Perhaps move the project null check to here?

Or simply not suppress, and just not add to the collection.